bin/dotc

```diff
@@ -57,11 +57,21 @@ function checkjar {
     else
       echo "The required jar file was built successfully."
     fi
+  else
+    NEW_FILES="$(find "$DOTTY_ROOT/$3" -iname "*.scala" -newer "$1")"
+    if [ ! -z "$NEW_FILES" ]; 
+    then
+      echo "new files detected. rebuilding"
+      cd $DOTTY_ROOT
+      sbt $2
+      touch "$1"
```

Comment:
Why do you need to do this manually? At a first sight, sbt should update the jar file when rebuilding, right?